src/test/ui/consts/dangling-alloc-id-ice.stderr

```diff
@@ -0,0 +1,13 @@
+error: this constant cannot be used
```

Comment:
This error message has been changed to "any use of this value will cause an error" by #53821.